### PR TITLE
filter changed

### PR DESCRIPTION
**Pull request by derins**
# Current issues
n/a

# Code changes
1. removed farm_device_id 16 for energy summary page 
2. checks if dict value is none
3. 15 minute intervals

# Documentation changes
none

# Validation suggestions
1. run farm account and load energy_summary_page

### CODE REVIEW

File: portal/app/api/v1/endpoint/farm_data_parser.py [hunk 1]
```diff
@@ -45,7 +45,9 @@ def local_fan_info_parser(data):
         Parses json blob for local fan info page and returns the argument fields
         as individual dictionaries in json form
     """
-    farm_data = data[0].get("device_data")
+    with open('response.json') as json_file:
```
Comment (jongoncalves) [inline portal/app/api/v1/endpoint/farm_data_parser.py:48]: Can you explain what this is doing. Where and when does this file get written? Why use a file and are you sure it'll exist?

File: portal/app/api/v1/endpoint/farm_data_parser.py [hunk 2]
```diff
@@ -67,9 +68,13 @@ def main_power_parser(data):
         Parses json blob for device_id = 17 and returns processed data
         Converts power from W to kW and adds field energy to the dataframe
     """
-    farm_data = [ main_farm_parser(row) for row in data ]
-    farm_df = pd.DataFrame(farm_data)
+    farm_data = []
+    for row in data:
+        row_val = main_farm_parser(row)
+        if row_val != None:
```
Comment (jongoncalves) [inline portal/app/api/v1/endpoint/farm_data_parser.py:74]: Do some digging as to why you should be using `is not` v.s. `!=`. Let me know what you find.

Comment (derins) [reply portal/app/api/v1/endpoint/farm_data_parser.py:74]: Equality and identity operators and “==” operator compares the values where as “is” operator checks to see if the memory location is the same, therefore the is operation is fundamentally faster. In python there can only be a single none object and therefore all the none values are pointing to the same memory address, which is the reason None should always be compared by the identity operation.

Comment (derins) [reply portal/app/api/v1/endpoint/farm_data_parser.py:74]: I just corrected it.

**Pull request merged**